task/handler/core/src/main/java/io/knotx/fragments/task/handler/consumer/EventLogConverter.java

```diff
@@ -117,16 +126,20 @@ private JsonObject getNodeLog(List<EventLogEntry> logs) {
         .findFirst();
   }
 
+  private Optional<EventLogEntry> getErrorsForExecution(List<EventLogEntry> logs) {
+    return logs.stream()
+        .filter(log -> ERROR_TRANSITION.equals(log.getTransition()))
+        .reduce((previous, current) -> current);
```

Comment:
Is it possible that there will be more than one `EventLogEntry` of type `error` logged for a given node?

Reply:
No, every operation is invoked only once by task engine. Using Actions, However, you can wrap an operation with some extra behaviour logic e.g circuit breaker with retry and fallback configured.

Reply:
It means that an operation can:
- end with success transition
- end with error transition (with error details in node log)
- throw an error (the task engine handles the error, sets error transition in the fragment result and adds the error to event log)